src/Fixtures/index.js

```diff
@@ -128,7 +964,7 @@ export default {
       equipment: 'One disc, 2 cones, 3 players',
       durationInMinutes: 18,
       goals: ['Offense', 'Handling'],
-      level: 'beginner',
+      level: 'Beginner',
```

Comment:
I would suggest using enums for all variables that have a fixed set of values. It prevents difference in Spelling, makes it a lot easier to figure out where something is used and makes auto completion work a lot better

https://www.sohamkamani.com/blog/2017/08/21/enums-in-javascript/

Reply:
I think introducing enums is an awesome idea
But we should do this in another pull request since it requires a bit of refactoring